src/DPOT_SOLIDIFY/dpot_test.cpp

```diff
@@ -1,8 +1,5 @@
 #include <Arduino.h>
 #include <SPI.h>
-#include <SD.h>
-
-#define CHIP_SELECT_PIN_SD 4
 
```

Comment:
these things are not used here, so I delete them

Reply:
I did this before [this](https://jmfrancklab.slack.com/archives/CRY3EL1AP/p1680821111333079?thread_ts=1680819265.672089&cid=CRY3EL1AP) communication, which makes it apparent that maybe you wanted a functional file-writing mechanism.  But, I would not start from this.  I would first start by drawing a flowchart of the previous SD_binary code.  It's pretty obvious that the following will not work, because the file is never opened!

Reply:
I am asking for an example because I believe I can fix it if I just have and example of how the code works drawing a flow chart will take too much time I strongly disagree with that